tests/integration/steps/integration_steps.go

```diff
@@ -86,16 +86,31 @@ func getKubectlOptions(namespace string) *k8s.KubectlOptions {
 	return k8s.NewKubectlOptions("", "", namespace)
 }
 
-func deployCluster(t *testing.T, namespace, customValues string, helmValues map[string]string, upgrade bool) {
+func installK8ssandraHelmRepo(t *testing.T) {
+	helm.RunHelmCommandAndGetOutputE(t, &helm.Options{}, "repo", "add", "k8ssandra", "https://helm.k8ssandra.io/stable")
+	helm.RunHelmCommandAndGetOutputE(t, &helm.Options{}, "repo", "update")
+}
+
+func deployCluster(t *testing.T, namespace, customValues string, helmValues map[string]string, upgrade bool, useLocalCharts bool, version string) {
 	clusterChartPath, err := filepath.Abs("../../charts/k8ssandra")
 	g(t).Expect(err).To(BeNil())
 
+	if !useLocalCharts {
+		installK8ssandraHelmRepo(t)
+		clusterChartPath = "k8ssandra/k8ssandra"
+	}
+
 	customChartPath, err := filepath.Abs("charts/" + customValues)
 	g(t).Expect(err).To(BeNil())
 
 	if os.Getenv("K8SSANDRA_CASSANDRA_VERSION") != "" {
-		log.Println(Info(fmt.Sprintf("Using Cassandra version %s", os.Getenv("K8SSANDRA_CASSANDRA_VERSION"))))
-		helmValues["cassandra.version"] = os.Getenv("K8SSANDRA_CASSANDRA_VERSION")
+		if !useLocalCharts && strings.HasPrefix(os.Getenv("K8SSANDRA_CASSANDRA_VERSION"), "3.11") {
```

Comment:
We can fix this later, but probably we shouldn't do it for 4.0 either. Although, also - this is a scenario which can fail in the real update.